### PR TITLE
Module1: My project initial Set up.

### PR DESCRIPTION
**Pull request by dunnyk**
This is my initial project set-up.
- Add all the index.html file.
- Add style.css file.
- Add image folder with all images used in this project.
- Detailed README.md file that says what the project is all about and how to spin this project in your local development.

### CODE REVIEW

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @dunnyk , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ]  Doesn't have a descriptive title 
- [ ] Doesn't have a meaningful short summary 
- [ ] Doesn't have a good commits

- [ ] Pleast try to revisit this [rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) and make the changes 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    <link rel="stylesheet" href="style.css">
+    <title>mobile version</title>
+</head>
+
+<body>
+
+    <header class="header">
+        <a class="container">Arrow Dan</a>
+        <nav class="navbar"></nav>
+            <img class="hamburger hide-desktop" src="image/Union.png" alt="navigation icon" />
```
Comment (zainsadaqat) [inline index.html]: Please wrap `<a>` tag and `` tag with `<nav>` tag.

**Comment by dunnyk** [comment]
Hi @[youmari](https://github.com/youmari), I did the changes you asked for. Kindly check.

**Review by zainsadaqat — CHANGES_REQUESTED** [review]
# Hello @dunnyk  😃 😄 

You're doing a great job on your project. We're proud of you. Good job so far! 
# 👏🏻 👏🏻 👏🏻 

There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
  ✅  Correct use of Github Flow
  ✅  No linter errors
  ✅  Descriptive Pull Request

## Required Changes ♻️

- [ ] Please remove the unnecessary space from its surroundings. 👍🏻 
Screen Shot 2022-04-21 at 10 58 24 AM

_Check the comments under the review._




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dunnyk** [comment]
hi @[zainsadaqat](https://github.com/zainsadaqat) have done the changes asked. Kindly check.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>mobile version</title>
+</head>
+
+<body>
```
Comment (aimemalaika) [inline index.html]: - [ ] Kindly add CSS to remove the padding and margin around the body of your page

File: index.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+    <header class="header">
+        <nav class="navbar">
+            <a class="container">Arrow Dan</a>
+            <img class="hamburger hide-desktop" src="image/Union.png" alt="navigation icon" />
+        </nav>
```
Comment (aimemalaika) [inline index.html]: - [ ] 1. Kindly set the hamburger as a  button

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @dunnyk , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

the-simpsons-homer-simpson

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (dunnyk) [reply index.html]: @[aimemalaika](https://github.com/aimemalaika) the hamburger thing is displayed as icon in figma... or maybe i did not get you kindly...

**Review by m05e5 — APPROVED** [review]
# Hi @dunnyk  , 

### Status: Approved ✔️
goodjob


:+1: You really did a great job. You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[Highly Recommended]



**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dunnyk** [comment]
Hi @[m05e5](https://github.com/m05e5), Thanks for your encouraging word. I must strive to become the developer I want to be globally. Success too.

**Comment by dunnyk** [comment]
Thanks caleb... happy to proceed to the next step.

**Pull request merged**